File: Wp.CIS.LynkSystems.WebApi/Models/Administrative/Product/ProductLookupValuesModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using Wp.CIS.LynkSystems.Model.Lookup;

namespace Wp.CIS.LynkSystems.WebApi.Models.Administrative.Product
{
    /// <summary>
    /// 
    /// </summary>
    public class ProductLookupValuesModel
    {
        /// <summary>
        /// 
        /// </summary>
        public IEnumerable<DownloadTimeModel> DownloadTimes { get; set; }

        /// <summary>
        /// 
        /// </summary>
        public IEnumerable<ProductTypeModel> ProductTypes { get; set; }

        /// <summary>
        /// 
        /// </summary>
        public IEnumerable<ManufacturerModel> Manufacturers { get; set; }

        /// <summary>
        /// 
        /// </summary>
        public IEnumerable<InstallTypeModel> InstallTypes { get; set; }

        /// <summary>
        /// 
        /// </summary>
        public IEnumerable<MobileLookupModel> MobileLookups { get; set; }

        /// <summary>
        /// 
        /// </summary>
        public IEnumerable<ProductBrandModel> Brands { get; set; }


    }
}

File: Wp.CIS.LynkSystems.Web/ClientApp/app/models/administration/product/productmaintenance/product-lookup-values.model.ts

import { ProductBrandModel } from './product-brand.model';
import { DownloadTimeModel } from './download-time.model';
import { ManufacturerModel } from './manufacturer.model';
import { ProductTypeModel } from './product-type.model';
import { InstallTypeModel } from './install-type.model';
import { MobileLookupModel } from './mobile-lookup.model';

export class ProductLookupValuesModel {

 
    public downloadTimes: Array<DownloadTimeModel>;

 
    public productTypes: Array<ProductTypeModel>;

 
    public manufacturers: Array<ManufacturerModel>;


    public installTypes: Array<InstallTypeModel>;

 
    public mobileLookups: Array<MobileLookupModel>;

 
    public brands: Array<ProductBrandModel>;


}